app/src/hooks/useTooltipOffset.ts

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from "react";
+
+export function useTooltipOffset(cardRef) {
+  const [offset, setOffset] = useState<[number, number]>([0, -130]);
+  const getOffset = () => {
```

Comment:
It's not exactly clear from this function and `useTooltipOffset` names, but the offset is a coordinate and that might be useful to know for whoever calls this function.